android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/PwoDiscoveryService.java

```diff
@@ -134,22 +155,30 @@ public void onCreate() {
   @Override
   @SuppressWarnings("deprecation")
   public int onStartCommand(Intent intent, int flags, int startId) {
-    // Since sometimes the lists have values when onStartCommand gets called
-    initializeLists();
-    // Start scanning only if the screen is on
-    PowerManager powerManager = (PowerManager) getSystemService(Context.POWER_SERVICE);
-    // NOTE: use powerManager.isInteractive() when minsdk >= 20
-    if (powerManager.isScreenOn()) {
-      startSearchingForPwos();
-    }
-
+    startSearchingForPwos();
     //make sure the service keeps running
     return START_STICKY;
   }
 
   @Override
   public IBinder onBind(Intent intent) {
-    throw new UnsupportedOperationException("Not yet implemented");
+    mIsBound = true;
+    return mBinder;
+  }
+
+  @Override
+  public boolean onUnbind(Intent intent) {
+    mIsBound = false;
+    if (mSecondScanComplete) {
```

Comment:
Yes.  Imagine we wake up, start the app, stop the app.  We will kill the service before it's had a chance to run 10 seconds and surface all the notifications.